sim/rogue/runes.go

```diff
@@ -83,10 +83,15 @@ func (rogue *Rogue) applyFocusedAttacks() {
 			aura.Activate(sim)
 		},
 		OnSpellHitDealt: func(aura *core.Aura, sim *core.Simulation, spell *core.Spell, result *core.SpellResult) {
-			if !spell.ProcMask.Matches(core.ProcMaskMeleeOrRanged) || !result.DidCrit() {
+			isFoKOH := false
+
+			if spell.ActionID.SpellID == 409240 && spell.ActionID.Tag == 2 {
+				isFoKOH = true
+			}
```

Comment:
Rather than do this, in the FOK code you could assign the FOK MH/OH spells to the rogue as properties, then just do `spell == rogue.FanOfKnivesOH`